flutter-studio/flutter-studio.iml

```diff
@@ -40,5 +39,6 @@
     <orderEntry type="module" module-name="intellij.android.common" />
     <orderEntry type="module" module-name="intellij.android.projectSystem" />
     <orderEntry type="module" module-name="intellij.android.projectSystem.gradle" />
+    <orderEntry type="module" module-name="intellij.android.adt.ui" />
```

Comment:
looks like this dependency was accidentally removed. not sure if it can be removed as it is now in the other xml file.